frontend/spec/controllers/spree/checkout_controller_spec.rb

```diff
@@ -168,6 +168,26 @@ def spree_post_address
             expect(order.ship_address.id).to eq(@expected_ship_address_id)
           end
         end
+
+        context "with zip codes with leading and trailing spaces" do
+          before do
+            spree_post :update, {
+                :state => "address",
+                :order => {
+                    :bill_address_attributes => order.bill_address.attributes.except("created_at", "updated_at").merge("zipcode": " 12345 "),
```

Comment:
Line is too long. [141/100]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.